scripts/election.py

```diff
@@ -190,15 +198,16 @@ def sort_key(x):
             reminder = copy.deepcopy(date)
             reminder["type"] = "reminder"
             reminder["deadline_date"] = reminder["date"]
-
+            friendly_date = pendulum.instance(reminder["date"]).format("MMMM Do")
+            reminder["explanation"] = f"{item.capitalize()} must be received by {friendly_date}."
 
             mail = copy.deepcopy(reminder)
             mail["date"] = reminder["date"] - one_day - one_week
-            mail["name"] = f"Mail {item}! It must be received"
+            mail["name"] = f"Mail {item}"
             reminders.append(mail)
 
             post_office = copy.deepcopy(reminder)
-            post_office["name"] = f"Mail {item} at the post office! It must be received"
+            post_office["name"] = f"Mail {item} at the post office"
```

Comment:
I'm not completely following this. I assume someplace else the date and the "It must be received" are merged into a string on a calendar item and its not missing here.

Reply:
Ya, it's moved to the "explanation" key. "name" needs to be a phrase so that it can be combined with other reminders.